src/api/handlers/twilio/postReply.js

```diff
@@ -1,15 +1,157 @@
+import moment from "moment";
+import { db } from "../../../../knex";
 import TemplatesModel from "../../../models/templates";
 import { getMessageResponse } from "../../../services/twilioClient";
 
-export default (req, res) => {
+const convertReply = (reply) => {
+  const languageConversions = {
+    yes: ["نعم", "አዎ", "haa", "Evet", "si", "yes"],
+    // In spanish and english no is the same
+    no: ["لا", "አይ", "maya", "Hayır", "no"],
+    interpreter: [
+      "مترجم",
+      "አስተርጓሚ",
+      "turjubaan",
+      "çevirmen",
+      "interprete",
+      "interpreter",
+    ],
+  };
+  let convertedReply;
 
-  const language = "English";
+  for (let key of Object.keys(languageConversions)) {
+    if (languageConversions[key].includes(reply)) {
+      convertedReply = key;
+      break;
+    }
+  }
 
-  const replyText = TemplatesModel.generateReply(language);
+  return convertedReply;
+};
+
+export async function getAppointments(patientPhoneNumber) {
+  return await db("appointments")
+    .select("appointmentId", "language")
+    .where({ patientPhoneNumber, appointmentIsConfirmed: 0 });
+}
+
+export async function handlePostReply(patientPhoneNumber, messageFromPatient) {
+  let ourResponse;
+
+  if (!messageFromPatient || messageFromPatient === "") {
+    ourResponse = getMessageResponse(
+      "We are sorry, we could not process your response. Please call Sanctuary Refugee Health Centre (Dr. Michael Stephenson) at 226-336-1321"
+    );
+    return ourResponse;
+  }
+
+  const lowercaseMessageFromPatient = messageFromPatient.toLowerCase();
```

Comment:
It'd good to trim whitespaces from the `messageFromPatient`, as it's common that phone keyboards add spaces